### ISSUE TITLE
Update CI config

### ISSUE THREAD

**Issue by jsumners** [pinojs/pino#1749]
+ [ ] Remove unsupported Node versions
+ [ ] Clean up any outdated conditional logic around Node 14

**Comment by cjreads665** [comment]
this is for the workflow file ci.yml correct?
And you want to ensure the ci pipeline can run the actions with node14 wholly removed from it?
at the same time, any dependency on node14 should be removed as well from the application code? @jsumners

**Comment by jsumners** [comment]
> this is for the workflow file ci.yml correct?

Yes, and any other workflows that reference Node versions to be removed from support (it should be 14 and 16).

> And you want to ensure the ci pipeline can run the actions with node14 wholly removed from it?

It's not about ensuring they run. It's really just a clean up job. We know newer versions work.

> at the same time, any dependency on node14 should be removed as well from the application code?

Yes, that is appropriate.

-----

For what it's worth, I really created this issue as a TODO task to complete before the release. I figured it would be a maintainer that takes care of it, which is why I didn't put a "good first issue" label on it. But you are certainly welcome to work on it if you like. Any PR should target the `next-v9` branch.

**Comment by cjreads665** [comment]
just a quick question - how do i run and debug it locally?
I'm sorry if it's a silly question but I have mostly worked on frontend applications where i basically run the code and access the app in the browser. I read the readme and was not able to find anything regarding that.
If it is not obvious, maybe i can add that to readme?
